docs/frontend-shared.md

```diff
@@ -1,67 +1,39 @@
 # Frontend Shared Package
 
-
 The `frontend-shared` package is a library of shared utilities, components and styles common to multiple Hypothesis front-end applications.
 
-The `frontend-shared` package source is located in `./frontend-shared` in this repository.
-
-Note: The long-term goal is to move this package into its own repository. There are some short-term advantages to the current setup: it piggy-backs on existing tooling and gives developers the benefit of working with just one repository when making changes (which primarily benefit the `client` in the first place).
-
-## How it works
-
-Frontend-shared works by creating a symlink via `$ yarn link` between the working dir
-`./frontend-shared/` and the installed dir `node_modules/@hypothesis/frontend-shared/`.
-
 ## Client setup
 
-To set up `frontend-shared` in the client, run
+To set up ``frontend-shared`` in the client, first build the package and then publish to a your local yalc repository. You may need to install yalc first. ``yarn global add yalc ``
 
 ```shell
-$ make dev
+# from the frontend-shared dir
+$ yarn build
+$ yalc publish
 ```
 
-as normal for development in the `client` application.
-
-This will trigger the `frontend-shared` build which:
-- Builds transpiled JSX into the `frontend-shared/lib` directory
-- Creates a linked package in the `node_modules/@hypothesis` folder (this is linked to `frontend-shared/lib`)
-
-When `$ make dev` is running, file changes will automatically trigger a rebuild. To rebuild manually, run:
+In the ``client`` repsitory, link to you local pushed reposotry
 
 ```shell
-$ gulp setup-frontend-shared
+# from the client dir
+$ yalc add  @hypothesis/frontend-shared
 ```
 
-## Usage in other repositories
+This will install the built local version of your ``frontend-shared`` respoitory in the ``client`` without a using symlink.
 
-If you have modified the `frontend-shared` package locally and would like to preview changes in another Hypothesis applications that imports the package--that is, if you want to test uncommitted changes in `frontend-shared` in another local repository before publishing changes--follow these steps:
+_Note_: Currently, there is no support for watchers so any changes made to the ``frontend-shared`` files will need to be rebuilt and re-pushed as follows
 
-1. In this repository's directory
 ```shell
-    $ gulp setup-frontend-shared
+# from the frontend-shared dir
+$ yarn build
+$ yalc push
```

Comment:
I think it would be good to note above that `yalc push` will update the copy of the package in all repositories that have `@hypothesis/frontend-shared` yalc-linked.

Reply:
I agree